tests/build.go

```diff
@@ -139,6 +140,53 @@ func testImageIDOutput(t *testing.T, sb integration.Sandbox) {
 	require.Equal(t, dgst, digest.Digest(md.ConfigDigest))
 }
 
+func testBuildMobyFromLocalImage(t *testing.T, sb integration.Sandbox) {
+	if sb.Name() != "docker" && sb.Name() != "docker+containerd" {
```

Comment:
Can we do the same `strings.Cut` trick here, so we don't need to hardcode the `containerd` part of the string?

Reply:
Perfect, LGTM :tada: